### DISCUSSION TITLE
query node 搜索期间机器负载异常飙高

### DISCUSSION THREAD

**Discussion by gy7779** [milvus-io/milvus#28714] [Q&A and General discussion]
Milvus 版本 2.3.3，集群部署
机器配置 CPU 40 核，256G 内存，部署 querynode * 2、querycoord * 1

Collection 分片：2
向量配置：256 维，metric_type:IP，nlist:16384，m:32，nbits:8
索引数据量：5600w

搜索配置：
```java
                    SearchParam param = SearchParam.newBuilder()
                            .withCollectionName("demo")
                            .withConsistencyLevel(ConsistencyLevelEnum.BOUNDED)
                            .withMetricType(MetricType.IP)
                            .withTopK(1000)
                            .withVectors(vectors)
                            .withVectorFieldName("vector")
                            .withParams("{\"nprobe\":32}")
                            .build();
                    R<SearchResults> r = milvusServiceClient.search(param);
```
线程数：5
期间单个 querynode 占用 cpu 15 核左右：
搜索期间机器 CPU：
![image](https://github.com/milvus-io/milvus/assets/89443035/4870e497-5dd5-499f-baea-58298b812755)
搜索期间机器负载：
![image](https://github.com/milvus-io/milvus/assets/89443035/88f7af9f-ae19-4d92-927c-0cdd9e49e56a)

在 5 线程的搜索测试下机器 cpu 基本打满，1 分钟负载达到了 150 且没有下降趋势，我想知道出现这种异常现象的原因及解决方法。

**Comment by xiaofan-luan** [comment]
这并不异常，Milvus会充分使用CPU资源进行并发。

如果查询性能很低（比如只有几十），可能才需要持续看。应该能达到几百qps
